internal/display/logs.go

```diff
@@ -160,23 +160,28 @@ func (r *Renderer) LogList(logs []*management.Log, ch <-chan []*management.Log,
 		res = append(res, &logView{Log: l, silent: silent, raw: l})
 	}
 
-	var viewChan chan View
+	r.Results(res)
+}
+
+func (r *Renderer) LogTail(logs []*management.Log, ch <-chan []*management.Log, silent bool) {
```

Comment:
The LogTail doesn't seem to attach at the top the table headers like the auth0 logs list does:

```
  TYPE                     DESCRIPTION                                             DATE                  CONNECTION            CLIENT
```